src/serverdlgbase.ui

```diff
@@ -352,23 +332,11 @@
   <tabstop>lvwClients</tabstop>
   <tabstop>tabWidget</tabstop>
   <tabstop>cbxDirectoryType</tabstop>
-  <tabstop>edtServerName</tabstop>
-  <tabstop>edtLocationCity</tabstop>
-  <tabstop>cbxLocationCountry</tabstop>
   <tabstop>chbEnableRecorder</tabstop>
   <tabstop>edtCurrentSessionDir</tabstop>
   <tabstop>pbtNewRecording</tabstop>
   <tabstop>tedWelcomeMessage</tabstop>
   <tabstop>cbxLanguage</tabstop>
-  <tabstop>pbtRecordingDir</tabstop>
-  <tabstop>edtRecordingDir</tabstop>
-  <tabstop>tbtClearRecordingDir</tabstop>
-  <tabstop>edtCustomDirectory</tabstop>
-  <tabstop>pbtServerListPersistence</tabstop>
-  <tabstop>edtServerListPersistence</tabstop>
-  <tabstop>tbtClearServerListPersistence</tabstop>
-  <tabstop>chbStartOnOSStart</tabstop>
-  <tabstop>chbEnableDelayPanning</tabstop>
  </tabstops>
```

Comment:
I think this should stay the same as before. I remember QtDesigner removes tab-stops automatically leading to accessibility problems.

Reply:
I don't edit .ui files by hand.

Reply:
It still needs to be fixed somehow

Reply:
If they're meant to be tabstops, there should be a way of marking them as such.  Just adding `<tabstop>` tags by hand isn't the way.

Reply:
Aha!

It must have been because I moved the fields into different containers... Strange it does that.  OK, fixed it.